<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DailyTasks</class>
 <widget class="QWidget" name="DailyTasks">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>346</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>620</width>
    <height>346</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>620</width>
    <height>346</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Daily Tasks</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0,0">
   <property name="spacing">
    <number>20</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
     <property name="spacing">
      <number>20</number>
     </property>
     <item>
      <widget class="QCalendarWidget" name="calendarWidget">
       <property name="font">
        <font>
         <family>Segoe Print</family>
        </font>
       </property>
       <property name="selectedDate">
        <date>
         <year>2016</year>
         <month>8</month>
         <day>2</day>
        </date>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2,1,2,1,2,4">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="m_pbtnAdd">
       <property name="font">
        <font>
         <family>Segoe Print</family>
        </font>
       </property>
       <property name="text">
        <string>Add task</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_pbtnEdit">
       <property name="font">
        <font>
         <family>Segoe Print</family>
        </font>
       </property>
       <property name="text">
        <string>Edit task</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_pbtnDel">
       <property name="font">
        <font>
         <family>Segoe Print</family>
        </font>
       </property>
       <property name="text">
        <string>Delete task</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBarTask">
       <property name="value">
        <number>24</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_pbtnOK">
       <property name="font">
        <font>
         <family>Segoe Print</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>calendarWidget</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>m_pbtnOK</sender>
   <signal>clicked()</signal>
   <receiver>DailyTasks</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>583</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>462</x>
     <y>216</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
